packages/components/src/internal/components/entities/FindDerivativesButton.tsx

```diff
@@ -190,7 +190,7 @@ export const FindDerivativesMenuItem: FC<Props> = memo(props => {
 
     return (
         <DisableableMenuItem
-            operationPermitted={!invalidFilterNames}
+            disabled={invalidFilterNames !== ''}
```

Comment:
can invalidFilterNames be undefined?

Reply:
No because it's calculated from an `array.join(',')` on an array that is always defined and defaults to `[]`.